egs2/slue-voxpopuli/asr1/conf/tuning_wavlm/train_asr_conformer_lr2e-3_warmup5k_wav2vec2_conv2d2_wdecay1e-5_bert_1.yaml

```diff
@@ -0,0 +1,118 @@
+specaug: specaug
```

Comment:
It would be better if you could combine some of the configs. It seems to me that many of them are very similar.

Reply:
Done.